### ISSUE TITLE
Split package

### ISSUE THREAD

**Issue by MarDiehl** [conda-forge/damask-feedstock#50]
DAMASK for most (or even all) other package managers is distributed as three packages (python-damask, damask-grid, damask-mesh) and one meta package.
That seems to work also for CONDA.

**Comment by jan-janssen** [comment]
Yes, here is an example to create multiple outputs for one package:
https://github.com/conda-forge/fenics-feedstock/blob/master/recipe/meta.yaml
